data/data-pipeline/data_pipeline/score/score.py

```diff
@@ -0,0 +1,14 @@
+import pandas as pd
+import data_pipeline.score.field_names as FN
+
+from data_pipeline.utils import get_module_logger
+
+logger = get_module_logger(__name__)
```

Comment:
We don't need the logger here if it's not used within the file.

Reply:
It's used in all of the child classes. I could redefine it in every child class, but that seems less DRY and also the more we move out of this parent class to every child class, the less it feels like we're utilizing the inheritance structure (which, admittedly, maybe we don't need).

Reply:
moved to child classes